### PR TITLE
Add schedule for daily DB backup

### PR DESCRIPTION
**Pull request by johnake**
### Context

Add schedule for daily DB backup

### Changes proposed in this pull request

This is to add a cronjob that triggers the backup and restore workflow at 02:00 UTC daily

### Guidance to review

Include any useful information needed to review this change.
Include any dependencies that are required for this change.

### Checklist

- [ ] Attach to Trello card
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

**Comment by johnake** [comment]
[Enable daily database refresh from PaaS to AKS](https://trello.com/c/Ceb7BYOx/365-enable-daily-database-refresh-from-paas-to-aks)

File: terraform/aks/application.tf [hunk 1]
```diff
@@ -33,11 +33,28 @@ module "application_configuration" {
...
+  namespace    = var.namespace
+  environment  = local.environment
+  service_name = local.service_name
+  cluster_configuration_map = module.cluster_data.configuration_map
+  kubernetes_config_map_name = module.application_configuration.kubernetes_config_map_name
+  kubernetes_secret_name     = module.application_configuration.kubernetes_secret_name
+  docker_image           = var.paas_app_docker_image
+  command       = ["bundle", "exec", "sidekiq", "-C", "./config/sidekiq.yml"]
+  probe_command = ["pgrep", "-f", "sidekiq"]
+  max_memory             = var.memory_max
```
Comment (RMcVelia) [inline terraform/aks/application.tf:58]: this should be worker specific e.g. var.worker_memory_max

File: terraform/aks/application.tf [hunk 2]
```diff
@@ -33,11 +33,28 @@ module "application_configuration" {
...
+  replicas               = var.replicas
```
Comment (RMcVelia) [inline terraform/aks/application.tf:59]: this should be worker specific e.g. var.worker_replicas

File: .github/workflows/database-copy.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    - cron: "0 2 * * *" # 02:00 UTC
+
+jobs:
+  restore_db:
+    name: Restore DB
+    strategy:
+      matrix:
+        environment: [dev]
+      max-parallel: 1
+    uses: ./.github/workflows/restore-paas-db-to-aks.yml
```
Comment (RMcVelia) [inline .github/workflows/database-copy.yml:15]: believe that means we need workflow_call added to restore-paas-db-to-aks.yml
i.e.
on:
  workflow_call:
    inputs:
      environment:
        type: string

**Review by RMcVelia — APPROVED** [review]

**Pull request merged**